### DISCUSSION TITLE
R ospsuite package installation

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1248] [Questions & Problems] [answered]
Hi

I am trying to call ospsuite package in R, but  I am getting this error message although I can see the package in the installed packages library. 
![image](https://user-images.githubusercontent.com/61779799/178227942-82c5b522-91ec-4289-83dc-52154ca50cfc.png)
![image](https://user-images.githubusercontent.com/61779799/178229309-30f70efa-27ee-43d3-a9dc-8abdf675d721.png)

I am using R V 4.2.0. I have installed the required dependent packages. For rClr I installed [Version 0.9.2 - R4]( https://github.com/Open-Systems-Pharmacology/rClr/releases/tag/v0.9.2). I tried earlier to install [Version 0.9.1 - R4](https://github.com/Open-Systems-Pharmacology/rClr/releases/tag/v0.9.1), but it did not work

![image](https://user-images.githubusercontent.com/61779799/178230158-a354903e-e916-465a-9bcf-b1f70238fd94.png)

Any clue, what is missing here?

**Comment by msevestre** [comment, marked as answer]
The message is asking you to install `ospsuite.utils`
Please follow the instructions on installation of  dependencies  for the package and you should be good to go

**Reply by bahsa505** [reply]
Thanks for your reply.

I have followed though the documentation.  I can't see ospsuite.utils at CRAN or a link to manually install it
![image](https://user-images.githubusercontent.com/61779799/178242615-92c3df80-7bc0-419b-9dd4-f765d28f9a4f.png)

**Comment by bahsa505** [comment]
It seems that the  `ospsuite.utils` installation link was not provided in the documentation. I found this [page](https://cran.r-project.org/web/packages/ospsuite.utils/index.html). 

The package was removed from the CRAN repository and the archived versions are not compatible with the current `ospsuite` package

![image](https://user-images.githubusercontent.com/61779799/178246533-6ad205db-e355-4d7e-9aa0-69e195f5d07b.png)


Is there any access to the recent version of ` ospsuite.utils`?

**Reply by msevestre** [reply]
Look at the instructions in the readme file
https://github.com/Open-Systems-Pharmacology/OSPSuite-R

If you scroll, there is a link to ospsuite.utils 1.3.17
Isn't this package working?

**Reply by bahsa505** [reply]
Thanks a lot, now it works.

I was looking at [OSPSuite-R](https://www.open-systems-pharmacology.org/OSPSuite-R/), but it doesn't seem to be updated with all package dependencies as the one you kindly provided.
